buildpack.md

```diff
@@ -238,15 +239,18 @@ The lifecycle MUST also store the Layer Content Metadata associated with each la
 Before a given re-build:
 - If a launch layer is marked `cache = false` and `build = false` in the previous image metadata, the lifecycle:
   - MUST restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `[types]` table.
+  - MUST restore any provided standardized Bill of Materials to `<layers>/<layer>.bom.<ext>`.
```

Comment:
We should note that we are only restoring BOM associated with layers instead of any BOM files.

Reply:
Please see 41d7cfd